plugins/guests/redhat/cap/nfs_client.rb

```diff
@@ -5,7 +5,11 @@ class NFSClient
         def self.nfs_client_install(machine)
           machine.communicate.sudo <<-EOH.gsub(/^ {12}/, '')
             if command -v dnf; then
-              dnf -y install nfs-utils nfs-utils-lib portmap
+              if $(output=$( dnf search nfs-utils-lib 2>&1) && grep -q 'No matches found' <<< "$output") ; then 
```

Comment:
Can we use `dnf info -q nfs-utils-lib > /dev/null` to test if the package exists in the repo? That would be better. BTW, if the guest OS's locale is not 'en' like, the output language of the `dnf` may not in English.

Reply:
I figured if I posted it, someone would tell me a better way to do the test :)